rpc/src/server.rs

```diff
@@ -1,32 +1,38 @@
 use crate::config::Config;
 use crate::module::{
-    ChainRpc, ChainRpcImpl, ExperimentRpc, ExperimentRpcImpl, IntegrationTestRpc,
-    IntegrationTestRpcImpl, MinerRpc, MinerRpcImpl, NetworkRpc, NetworkRpcImpl, PoolRpc,
-    PoolRpcImpl, StatsRpc, StatsRpcImpl,
+    AlertRpc, AlertRpcImpl, ChainRpc, ChainRpcImpl, ExperimentRpc, ExperimentRpcImpl,
+    IntegrationTestRpc, IntegrationTestRpcImpl, MinerRpc, MinerRpcImpl, NetworkRpc, NetworkRpcImpl,
+    PoolRpc, PoolRpcImpl, StatsRpc, StatsRpcImpl,
 };
 use ckb_chain::chain::ChainController;
 use ckb_miner::BlockAssemblerController;
 use ckb_network::NetworkController;
+use ckb_network_alert::{notifier::Notifier as AlertNotifier, verifier::Verifier as AlertVerifier};
 use ckb_shared::shared::Shared;
 use ckb_store::ChainStore;
 use ckb_sync::Synchronizer;
+use ckb_util::Mutex;
 use jsonrpc_core::IoHandler;
 use jsonrpc_http_server::{Server, ServerBuilder};
 use jsonrpc_server_utils::cors::AccessControlAllowOrigin;
 use jsonrpc_server_utils::hosts::DomainsValidation;
+use std::sync::Arc;
 
 pub struct RpcServer {
     pub(crate) server: Server,
 }
 
 impl RpcServer {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
It's better to refactor this. When the alert is disabled, why do we still need to provide `alert_notifier` and `alert_verifier`. The clippy linters all have their reason, we should not just disable them.

Reply:
We can split it into modules, with several `enable_xxx(xx_arg1, xx_arg2` methods, and use a `start` method to really start the service.